### ISSUE TITLE
support create view 

### ISSUE THREAD

**Issue by Luyang2902** [questdb/questdb#2201]
### Is your feature request related to a problem?

why not support view ?it's very important function to database.

**Comment by pswu11** [comment]
Hi @Luyang2902 please provide more context on the feature and how you intend to achieve. Thanks!

**Comment by puzpuzpuz** [comment]
Related to https://github.com/questdb/roadmap/issues/16

**Comment by korkmazozcan** [comment]
Could you please inform us about the current status of the development of this feature? Thanks!

**Comment by puzpuzpuz** [comment]
> Could you please inform us about the current status of the development and possible release date for this feature? Thanks!
> 
> P.S: I think pull request #4937 is the related PR

Yes, that's the PR. As for the status, this feature is in-the-works and should be shipped in the next few months.
